tooltester/pom.xml

```diff
@@ -40,7 +40,7 @@
         <system-stubs.version>2.0.1</system-stubs.version>
         <jackson.version>2.14.1</jackson.version>
         <aws.version>1.12.387</aws.version>
-        <dockstore.core.version>1.14.0-alpha.9</dockstore.core.version>
+        <dockstore.core.version>1.14.0-alpha.10</dockstore.core.version>
```

Comment:
Upgraded `dockstore.core.version` so that I can use the metric classes from this [PR](https://github.com/dockstore/dockstore/pull/5447).